tests/chains/ethereum/scripts/cmd/helper/contract.go

```diff
@@ -0,0 +1,37 @@
+package helper
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	IBCHandlerAddress              = "0x702E40245797c5a2108A566b3CE2Bf14Bc6aF841"
+	IBCCommitmentTestHelperAddress = "0x87d7778dbc81251D5A0D78DFD8a0C359887E98C9"
+	SimpleTokenAddress             = "0xff77D90D6aA12db33d3Ba50A34fB25401f6e4c4F"
+	ICS20TransferBankAddress       = "0xaE1C9125BbcF63bf51294C4D15CBD472782E330D"
+	ICS20BankAddress               = "0x2F5703804E29F4252FA9405B8D357220d11b3bd9"
+)
```

Comment:
I'm sorry if my previous comments caused any misunderstanding about settings.

Reply:
Move address to config json
https://github.com/dongrie/yui-relayer-build/blob/v0.3/tests/cases/eth2eth/configs/chains/ibc-0.json#L14-L18

https://github.com/dongrie/yui-relayer-build/blob/v0.3/tests/cases/eth2eth/configs/chains/ibc-0.json#L2-L9
The `"chain"` in the above cannot be inserted because it is a protobuf message.

Reply:
Let me put it simply: do not change the format of the existing configuration files (`path.json` or `ibc-[01].json`).
You may freely introduce a new config file for your CLI tools.
However, my recommendation is to pass all information by command-line arguments without using a new config file.

Reply:
The reason why you shouldn't change the format of the existing config files is that their format is defined in yui-relayer/core and it can't/shouldn't be changed.

Reply:
https://github.com/datachainlab/yui-relayer-build/pull/5/commits/5b8065e1b0649187b5d25f35e302de2475675f88

Removed address from json file and add to cli arguments